### PR TITLE
[fr] Documentation release v3.7.0 (#341)

### PR DESCRIPTION
**Pull request by jujudusud**
# Does this need translation?

- [ ] Yes 
- [X] No 

# Changes

### CODE REVIEW

File: wiki/fr/fr-Onboarding.md [hunk 1]
```diff
@@ -29,29 +35,34 @@ Maintenant vous allez voir ceci :
...
+Si vous ne voulez pas que les autres entendent votre audio, cliquez sur « Me silencer » (à gauche de votre mixage personnel) ce qui empêchera l'envoi de votre audio aux autres musiciens.
 
 Pendant que nous y sommes : « Paramètres » ouvre, évidement, la fenêtre des paramètres avec tous les paramètres audio que vous voudrez probablement modifier pour vos besoins.
 
 De plus amples informations sur l'utilisation de Jamulus peuvent être trouvées dans le [manuel du logiciel](Software-Manual).
 
 ## Avancé
-<!-- Bien sûr, vous pouvez faire beaucoup plus avec Jamulus, comme configurer un serveur privé, faire des concerts en ligne, … Jetez un coup d'œil sur ce site ! La communauté publie des guides, trucs, astuces dans la [base de connaissance](/kb/). --> 
+<!-- Bien sûr, vous pouvez faire beaucoup plus avec Jamulus, comme configurer un serveur privé, faire des concerts en ligne, … Jetez un coup d'œil sur ce site ! La communauté publie des guides, trucs et astuces dans la [base de connaissance](/kb/). -->
```
Comment (ann0see) [inline wiki/fr/fr-Onboarding.md:56]: Are you sure this is still commented?

**Review by ann0see — COMMENTED** [review]
Great! Thanks for updating it!

File: wiki/fr/fr-Onboarding.md (hunk 1 above)
Comment (jujudusud) [reply wiki/fr/fr-Onboarding.md:56]: Yes it's already translated but changing only "," by "et" sounds better for me.

**Comment by jujudusud** [comment]
In this one, I have a warning. I don't understand how to fix.

`reviewdog: Reporting results for "remark-lint"
Warning: [remark-lint] reported by reviewdog 🐶
Incorrect list-item indent: add 2 spaces              list-item-indent            remark-lint

Raw Output:
             19:3  warning  Incorrect list-item indent: add 2 spaces              list-item-indent            remark-lint`

Anybody can help me ?

**Comment by jujudusud** [comment]
It's understood and ok now. :-)

**Review by gilgongo — APPROVED** [review]

**Pull request merged**